pkg/documentlog/store/sql.go

```diff
@@ -69,11 +70,15 @@ func newSQLDocumentStore(db *gorm.DB) (DocumentStore, error) {
 	if err := docStore.migrate(); err != nil {
 		return nil, err
 	}
+	// Since the content hash column was added later, make sure it's populated
+	if err := docStore.updateContentHashes(); err != nil {
```

Comment:
I think the migrate can return a particular error indicating no change which you could use to migrate only once!
But this would require everything to be run in a single transaction.

Reply:
or mark as to be removed from 0.x?

Reply:
It's fine to have it run multiple times since if there's nothing to migrate, it won't mutate anything. I'll make an issue to remove it from next version + 1.